src/foam/u2/layout/DisplayWidth.js

```diff
@@ -0,0 +1,23 @@
+foam.ENUM({
+  package: 'foam.u2.layout',
+  name: 'DisplayWidth',
+  documentation: `
+      An enum of the responsive display width types
+  `,
+
+  values: [
+      { name: 'XXS', label: 1440, cols: 8 },
```

Comment:
You shouldn't use `label` for these values. `label` is typically used when we're rendering the enum in a dropdown and I think the default (which is just name) would be good enough for that value. Change the name of these properties to something like `minWidth` or `maxWidth`. I think I like `minWidth` better because then `XS` would just have a value of `0`. If you do `maxWidth`, `XL` would need a value of `Number.MAX_SAFE_INTEGER` or something which looks a little weird IMO.

Reply:
Done.